src/loader/loader.cr

```diff
@@ -27,7 +27,6 @@ module BakedFileSystem
       files.each do |path|
         io << "@@files << BakedFileSystem::BakedFile.new(\n"
         io << "  path:            " << path[root_path_length..-1].dump << ",\n"
-        io << "  mime_type:       " << (mime_type(path) || `file -b --mime-type #{path}`.strip).dump << ",\n"
```

Comment:
I think most users of this lib will use it in some kind of web server and why to implement something like this over and over again?
I just wonder how can you replace call of that system command `file -b ...`? Do you have some example? Because I don't know and I can't even google it how to solve that in runtime.